src/fparser/two/tests/test_utils.py

```diff
@@ -141,3 +144,30 @@ def test_endstmtbase_match():
         require_stmt_type=True,
     )
     assert result == ("SUBROUTINE", Fortran2003.Name("sub"))
+
+
+@pytest.mark.usefixtures("f2003_create")
+def test_base_include_handling(tmpdir):
```

Comment:
I'm also not clear why the test is here given that the changes are solely in `common/*`. I note the comment in the test but why can't a failing match and returned content can be tested in isolation, or fparser2 used as within an example, or exampls, in the reader tests?

Reply:
I've removed this test now.